setup.cfg

```diff
@@ -8,7 +8,7 @@
 
 [flake8]
 hang-closing = true
-ignore       = E128,E251,F403,F405,E501,W503
+ignore       = E128,E251,F403,F405,E501,W503,E127
```

Comment:
Could you indicate what is this error in the comment header ?

Reply:
I made a mistake, I was trying to solve another error 127 (not from flake8 finally). I cancel the change